FreeAPS/Sources/Modules/Home/HomeStateModel.swift

```diff
@@ -348,6 +350,21 @@ extension Home {
             }
         }
 
+        /// Display the eventual statut message provided by the manager of the pump
```

Comment:
change comment from statut to status

Reply:
@avouspierre , if you change this code comment from statut to status, I believe this PR can be merged.

Reply:
I made the change to the code comment and pushed to your branch, hope thats ok :-)